asm/cluster/cluster.yaml

```diff
@@ -31,17 +31,3 @@ spec:
     identityNamespace: default
   labels:
     mesh_id: "[PROJECT_ID]_us-central1-c_asm" # {"type":"string","x-kustomize":{"partialSetters":[{"name":"project-id","value":"[PROJECT_ID]"},{"name":"cluster-name","value":"asm"},{"name":"gcloud.compute.zone","value":"us-central1-c"}]}}
-  loggingService: logging.googleapis.com/kubernetes
-  monitoringService: monitoring.googleapis.com/kubernetes
-  addonsConfig:
-    horizontalPodAutoscaling:
-      disabled: false
-    httpLoadBalancing:
-      disabled: false
-    cloudrunConfig:
-      disabled: true # {"type":"boolean","x-kustomize":{"setter":{"name":"cloud-run-disabled","value":"true"}}}
-  ipAllocationPolicy:
-    useIpAliases: true
-  masterAuth:
-    username: admin
-  network: default
```

Comment:
It might make sense to leave some fields as commented out, so that people who do want to change things can do it more easily. e.g. we know people want to commonly set network/subnetwork, so I'd keep at least that (commented out).

Reply:
Added network/subnetwork. I don't think the rest need to be included.